docs/get-started/getting-started-angular.mdx

```diff
@@ -119,21 +119,25 @@ APP_CLIENT_SECRET=<app-client-secret>
   </TabItem>
   </Tabs>
 
---- 
+---
 
 ### API token variable
 
-In this step, you'll continue to work in the Admin Console to create an API token for your tenant.  
+In this step, you'll continue to work in the Admin Console to create an API token for your tenant.
 
-1. In the Admin Console, switch to the **Beyond Identity Admin** realm, select **Apps > Beyond Identity Management API**.
+1. In the Admin Console, switch to the **Beyond Identity Admin** realm by selecting **Go to realm > Beyond Identity Admin**.
+
+  ![Go To Beyond Identity Admin Realm](../images/go-to-bi-admin-realm.png)
+
+1. Select **Apps > Beyond Identity Management API**.
 
   ![Admin Console Beyond Identity Admin realm](../images/apps-beyond-identity-management-api.png)
 
-2. Select the **API Tokens** tab and click **Create a token**. 
+2. Select the **API Tokens** tab and click **Create a token**.
 
 3. Enter a name for the token and click **Create token**.
 
-4. In the confirmation message, copy the API token and click **Close this message**. 
+4. In the confirmation message, copy the API token and click **Close this message**.
```

Comment:
nit: We might as well convert these to `1`'s, like you did in `getting-started-react`, if only for consistency. The only time this is not useful is if an include also has a numbered list, which can throw off docusaurus's count.